helm/slurm-cluster-storage/templates/_helpers.tpl

```diff
@@ -106,6 +106,40 @@
 ---
 */}}
 
+{{/* Controller spool volume */}}
+{{- define "slurm-cluster-storage.volume.accounting.name" -}}
+    {{- required "Controller spool volume name is required." .Values.volume.accounting.name | trim | kebabcase -}}
+{{- end }}
+
+{{/* Controller spool PV name */}}
+{{- define "slurm-cluster-storage.volume.accounting.pv" -}}
+    {{- cat (include "slurm-cluster-storage.volume.accounting.name" .) "pv" | kebabcase | quote -}}
+{{- end }}
+
+{{/* Controller spool mount name */}}
+{{- define "slurm-cluster-storage.volume.accounting.mount" -}}
+    {{- cat (include "slurm-cluster-storage.volume.accounting.name" .) "mount" | kebabcase | quote -}}
+{{- end }}
+
+{{/* Controller spool storage class name */}}
+{{- define "slurm-cluster-storage.volume.accounting.storageClass" -}}
+    {{- include "slurm-cluster-storage.class.local.name" . -}}
+{{- end }}
+
+{{/* Controller spool size */}}
+{{- define "slurm-cluster-storage.volume.accounting.size" -}}
+    {{- required "Controller spool volume size is required." .Values.volume.accounting.size -}}
+{{- end }}
+
+{{/* Controller spool device name */}}
+{{- define "slurm-cluster-storage.volume.accounting.device" -}}
+    {{- required "Controller spool Filestore device name is required." .Values.volume.accounting.filestoreDeviceName | trim | kebabcase -}}
+{{- end }}
```

Comment:
Comments and `required` messages refer to controller spool instead of accounting